src/app/views/functions/_form.html.erb

```diff
@@ -18,6 +18,11 @@
     <div class="control"> <%= form.text_area :usage, rows: 5, class: :textarea %> </div>
   </div>
 
+  <div class="field">
+    <%= form.label :tags_list, class: :label %>
+    <div class="control"> <%= form.text_field :tags_list %> </div>
```

Comment:
we need a hit text under this input to tell the user to input a comma separated tags and state the maximum number of tags and the format

check bluma for the correct way to present this